### ISSUE TITLE
Add more paths settings for parts libraries in preferences

### ISSUE THREAD

**Issue by ghost** [leozide/leocad#744]
### Is your feature request related to a problem? Please describe.

Actually there is no way to use additional parts with built-in parts (as AppImage actually include latest release of LDraw Parts Library) ***without*** install `complete.zip`.

So, if I want use fresh Unofficial LDraw Part Library (`ldrawunf.zip`) I need to install Official LDraw Parts Libary (`complete.zip`) too, BUT `complete.zip` already is bundled in AppImage.

Also, actually there is no way to use own/private parts together with built-in parts & `ldrawunf.zip`.

### Describe the solution you'd like

There should be 3 paths setting in *"Preferences > Custom Paths"*:

1. *"Official Parts Library"* - it should already show path to installed distributed `library.bin` on start in Windows, macOS and Linux (+ in AppImage case it should show path to built-in parts as `/tmp/mount_<id>/usr/share/leocad/library.bin` (also, it probably should allow only paths which ends with `/library.bin` and `/complete.zip`)
2. *"Unofficial Parts Library"* - for connecting unofficial parts library (probably, only paths ended with `/ldrawunf.zip` should be allowed);
3. "Custom Parts Library" - for connecting own/private parts (as folder or as ZIP).

If all three paths are set correctly all three libraries should woks together.

### Describe alternatives you've considered

Alternatively next should be implemented:

1. If in *"Preferences > Custom Paths > Parts Library"* there is path ended with `/ldrawunf.zip` LeoCAD should load it as additional parts library, but not replacing bundled official parts set (i.e. both built-in parts & parts from `ldrawunf.zip` should be used together, at least for AppImage builds);
2. Add in *"Preferences > Custom Paths > Additional Parts Library"* path setting to connect additional parts libraries — it would allow use custom parts, which are not included in built-in & `ldrawunf.zip` libraries (so, user may use own/personal parts all together with built-in & `ldrawunf.zip`)
2.1. Also, I recommend add option to add more than one "Additional Parts Library" by inserting its paths in entry field separated with `;` symbol.

### Additional context

If put path to Unofficial parts (`ldrawunf.zip`) in *"Preferences > Custom Paths > Parts Library"* entry field then run AppImage, LeoCAD fully ignores built-in Official parts — as result only Unofficial parts are loaded and those Unofficial parts which depends on its

**Comment by j6t** [comment]
I also think there should be a possibility to specify an arbitrary number of parts libraries. But I also think that separating fields with `;` symbol is inappropriate; it is too Windows specific (on Linux we would use `:` instead). A list of paths would be much more discoverable for users.

**Comment by Symbian9** [comment]
> But I also think that separating fields with `;` symbol is inappropriate; it is too Windows specific (on Linux we would use `:` instead).

Parsing `;` as `:` separator could be done on Linux-specific code level internally in LeoCAD, so that would not be an issue here.

**Comment by j6t** [comment]
> Parsing `;` as `:` separator could be done on Linux-specific code level internally in LeoCAD, so that would not be an issue here.

Of course, it's possible. But it is still not discoverable. A list of paths is preferable, IMNSHO.

**Comment by rsbx** [comment]
The separator character doesn't really matter much as long as it can be 
escaped.

Discoverability can be solved by documentation (already lacking) or an 
editing "interface".

**Comment by Symbian9** [comment]
> A list of paths is preferable, IMNSHO.

Yep, that is would enough to solve this issue.
